src/main/java/az/edu/turing/domain/dao/FlightDao.java

```diff
@@ -2,10 +2,12 @@
 
 import az.edu.turing.domain.entities.FlightEntity;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 public abstract class FlightDao implements Dao<FlightEntity, Long> {
+
     public abstract FlightEntity updateAvailableSeats(long flightId, int newAvailableSeatCount);
 
-    public abstract List<FlightEntity> getAllFlightsWithin24Hours();
+    public abstract List<FlightEntity> getFlightsWithin24Hours(LocalDateTime now, LocalDateTime twentyFourHoursLater);
```

Comment:
this method shouldn't take parameter. Instead, you can use `LocalDateTime.now()` and `LocalDateTime.now().plusHour(24)` in the implementer class